lib/skate_web/components/layouts/app.html.heex

```diff
@@ -45,15 +45,9 @@
 
     <link rel="stylesheet" href={static_content_route(@conn, "/css/app.css")} />
 
-    <%= if record_fullstory?() do %>
-      <%= _fullstory(assigns) %>
-    <% end %>
-    <%= if record_appcues?() do %>
-      <%= _appcues(assigns) %>
-    <% end %>
-    <%= if record_sentry?() do %>
-      <%= _sentry(assigns) %>
-    <% end %>
+    <._fullstory :if={record_fullstory?()}/>
+    <._appcues :if={record_appcues?()}/>
+    <._sentry :if={record_sentry?()}/>
```

Comment:
**issue:** heads up, I noticed that CI failed and it appears to be because the formatter doesn't like this (it wants a space before the `/>` closing the tags).

Reply:
Sorry about that! I saw, I did it right before a meeting and maybe shouldn’t have. Will fix afterward.

Reply:
No worries! I just happened to notice it and wanted to make sure you had seen, no rush.